.github/workflows/build.yml

```diff
@@ -81,7 +75,6 @@ jobs:
     - name: Compress Server Binary
       id: compress_server_binary
       run: |
-        # We normalize windows+choco ghc version 8.10.2.2
```

Comment:
Which part was the normalization? Was it `GHC_VER=$(echo $GHC_VER | sed 's/8.10.2.2/8.10.2/g')` part?

Reply:
yep, not sure if 8.10.5 is in the choco package yet 🤔

Reply:
@jneira it's unfortunately not in the list... I added a comment on the package

Reply:
it still is not

Reply:
I also sent a mail to the maintainer. I hope this will be resolved soon...

Reply:
There is a [github repo](https://github.com/Mistuke/GhcChoco), i usually open issues about there

Reply:
Oh I didn't know that, I will make a PR

Reply:
@jneira Actually the maintainer was faster than my PR :) Now there's GHC 8.10.5